.github/workflows/ci.yml

```diff
@@ -61,9 +61,13 @@ jobs:
           key:
             cache|${{ runner.os }}|${{ env.PY }}|${{ hashFiles('pyproject.toml') }}|${{
             hashFiles('poetry.lock') }}|${{ hashFiles('.pre-commit-config.yaml') }}
-      - run: python -m pip install poetry poetry-dynamic-versioning
+      - name: Install poetry
         shell: bash
-      - run: poetry install
+        run: |
+          python -m pip install poetry poetry-dynamic-versioning
```

Comment:
Here you should add poethepoet right? So we can leverage our lovely brand new task manager to run the CI?

Reply:
good idea, how do you like it now?

I also removed the redundant poetry install in the build stage.